code/modules/client/preference/preferences.dm

```diff
@@ -2149,7 +2149,6 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 
 	if(disabilities & DISABILITY_FLAG_FAT && character.species.flags & CAN_BE_FAT)
 		character.dna.SetSEState(FATBLOCK,1,1)
-		character.mutations += FAT
```

Comment:
Setting FAT was made redundant by overeatduration, as overeatduration over a certain threshold triggers life() to automatically apply the FAT mutation on the mob. That's where the conflict arose, it was setting FAT twice and causing it to break-- making it such that the mob could never appear fat without clothes.

I tested it by setting up a Human character with the obesity disability selected and I became fat as soon as I spawned in when life() caught up to the fact that my mob's overeatduration was over the threshold

As for the OBESITY mutation being made redundant by the FATBLOCK being set, I'll remove that line too and see what effect it has, thanks for pointing this out